util/import.ts

```diff
@@ -10,18 +11,25 @@ import { TestCaseInfo } from '../state/atoms/patientTestCase';
  * valid desired populations
  * @returns An fqm-testify test case
  */
-export function bundleToTestCase(bundle: fhir4.Bundle, populationGroupCodes: string[]): TestCaseInfo {
+export function bundleToTestCase(
+  bundle: fhir4.Bundle,
+  populationGroupCodes: string[],
+  testCaseMap: Record<string, string | undefined>,
+  measureURL?: string
+): TestCaseInfo {
   if (!bundle.entry || bundle.entry.length === 0) {
     throw new Error('Bundle has no entries');
   }
 
   const patientEntry = bundle.entry.find(e => e.resource?.resourceType === 'Patient');
   const desiredPopulations: string[] = [];
-  const testCaseMeasureReportArr = bundle.entry.filter(isTestCaseMeasureReport);
+  const testCaseMeasureReportArr = bundle.entry.filter(
+    e => isTestCaseMeasureReport(e) && (e.resource as MeasureReport).measure === measureURL
+  );
   if (testCaseMeasureReportArr.length > 1) {
     // TODO: Once we have import errors persist on page, replace this!!!
     throw new Error(
-      `Expected 0 or 1 test case measure reports in bundle, but found ${testCaseMeasureReportArr.length}`
+      `Expected 0 or 1 test case measure reports in bundle that match the loaded measure, but found ${testCaseMeasureReportArr.length}`
```

Comment:
We may have to make this message a little clearer. Correct me if I am wrong, but I think we could still have 1 measure report in the bundle (theoretically) but that doesn't match the loaded measure? (I don't think we have actually seen this tho)

Reply:
My thought is that if there is a single test case MeasureReport in the bundle, we should use it for the expected values. But if there are multiple, then we should find the one that matches the Measure's url.